public/scripts/worktrollyUI.js

```diff
@@ -9,11 +9,29 @@ $("#createNewTaskButton").click(function() {
 
 });
 
+$("#createNewSprint").ready(function() {
+    $("#createNewSprint").hide(0);
+});
+
+$("#createNewSprintButton").click(function() {
+    $("#createNewSprint").show(100);
+    var date = new Date();
+    $("#creationDateCreateNewSprint").html(date.getDate() + "/" + (date.getMonth() + 1) + "/" +   date.getFullYear());
+});
```

Comment:
This method must use uiloader to load new page.

Reply:
The Date and other things must be removed and must be handled in backend (firebase functions)